db/migrations/20200504135835_remove_urn_state.sql

```diff
@@ -0,0 +1,226 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+DROP FUNCTION IF EXISTS api.all_urn_states;
+DROP FUNCTION IF EXISTS api.get_urn;
+
+DROP FUNCTION IF EXISTS api.bite_event_urn;
+CREATE OR REPLACE FUNCTION api.bite_event_urn(event api.bite_event) RETURNS api.urn_snapshot
+     LANGUAGE sql STABLE
+     AS $$
+SELECT * FROM api.urn_snapshot
+WHERE ilk_identifier = event.ilk_identifier
+    AND urn_identifier = event.urn_identifier
+    AND block_height <= event.block_height
+$$;
+
+DROP FUNCTION IF EXISTS api.flip_bid_snapshot_urn;
+CREATE FUNCTION api.flip_bid_snapshot_urn(flip api.flip_bid_snapshot) RETURNS api.urn_snapshot
+       LANGUAGE sql STABLE
+       AS $$
+SELECT * FROM api.urn_snapshot
+WHERE ilk_identifier = (SELECT identifier FROM maker.ilks WHERE ilks.id = flip.ilk_id)
+AND urn_identifier = (SELECT identifier FROM maker.urns WHERE urns.id = flip.urn_id)
+AND block_height <= flip.block_height
```

Comment:
I think this may need to `order by block_height desc limit 1` now that we're querying from a table that could have multiple rows with matching `urn_identifier`/`ilk_identifier`. Probably applies to queries below as well

Reply:
I restored get)urn - which should take care of that.